builder/builder.go

```diff
@@ -51,7 +51,7 @@ type eleLimit struct {
 // the value of _limit must be a slice whose type should be []uint and must contain two uints(ie: []uint{0, 100}).
 // the value of _having must be a map just like where but only support =,in,>,>=,<,<=,<>,!=
 // for more examples,see README.md or open a issue.
-func BuildSelect(table string, where map[string]interface{}, selectField []string) (cond string, vals []interface{}, err error) {
+func BuildSelect(table string, where map[string]interface{}, selectField []string, lock bool) (cond string, vals []interface{}, err error) {
```

Comment:
This line changed the `BuildSelect` exposed arguments, its a break change!
How about something like a magic word `_lock`?